root/static/scripts/timeline.js

```diff
@@ -212,13 +212,26 @@ class TimelineViewModel {
 
         _.forEach(parts, function (part) {
             var match;
-            if (match = part.match(/^(-)?([rv])-?$/)) { // trailing - for backwards-compatibility
+
+            /** Please correct me if i'm misunderstanding this
+             * if(match = part.match(RegExp) {
+             *    // Do sth!!
+             * }
```

Comment:
You can remove this entire comment now, hopefully we've clarified things here. :) (Next time feel free to ask questions like this on the pull request instead of adding them as code comments.)

Reply:
Oh thanksss i will ask through the pull request next time :D <3